### PR TITLE
test: Disable the gsheet test

### PR DESCRIPTION
**Pull request by sarojsarab**
## Description
- Disable gsheet tests

## Type of change
- Cypress

## How Has This Been Tested?
- Cypress

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js [hunk 1]
```diff
@@ -33,59 +33,60 @@ describe("Upgrade appsmith version", () => {
...
-      cy.forceVisit(testdata.APPURL);
-      agHelper.GetNClick(".t--widget-iconbuttonwidget button", 0, true);
-      agHelper.Sleep(1000);
-      agHelper.GetNAssertElementText(
-        ".tbody>div",
-        "1Developmentexpertise",
-        "have.text",
-        1,
-      );
+    // if (CURRENT_REPO === REPO.CE) {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js:38]: Have we moved to infra already? Why are we commenting it now?

Comment (sarojsarab) [reply app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js:38]: No its not yet moved to infra tests, this test has gotten unstable and failing a lot of time so better to disable it now.

Comment (Aishwarya-U-R) [reply app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js:38]: Still not a reason to remove. 
But why not - remove the whole spec to manual suite then? When validation is commented - why do stop/start container? Any reason?

Comment (sarojsarab) [reply app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js:38]: There are two verification happens here one is upgrade and another one is gsheet test, upgrade works fine and not at all flaky, so why to stop the upgrade test.

Comment (Aishwarya-U-R) [reply app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js:38]: How are we verifying the upgrade happened fine - if the gsheet is commented?

Comment (sarojsarab) [reply app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js:38]: We verify the logs if the container is started from docker logs.

Comment (Aishwarya-U-R) [reply app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js:38]: What does verifying logs confirm?

Comment (sarojsarab) [reply app/client/cypress/integration/Regression_TestSuite/UpgradeAppsmith/UpgradeAppsimth_spec.js]: Moved the test to manual suite

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**